contracts/deployment_scripts/testnet/layer1/001_deploy_tokens.ts

```diff
@@ -26,4 +34,3 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
 
 export default func;
 func.tags = ['HPERC20', 'HPERC20_deploy'];
-func.dependencies = ['ObscuroBridge'];
```

Comment:
Doesn't depend on the bridge any longer ? Or is the bridge deployed in another stage ?

Reply:
The whitelisting task depends on the bridge (to create wrapped tokens); This is just deploying erc20 on the L1.